project/Dependencies.scala

```diff
@@ -8,7 +8,7 @@ import scala.util.Try
 
 object Dependencies {
   val scala212 = "2.12.18"
-  val scala213 = "2.13.12"
+  val scala213 = sys.props.getOrElse("scala.nightly", "2.13.12")
```

Comment:
not sure how scala-steward will handle that, but since the bump is manual during the embargo, it does not matter much